test/Fixtures/ReflectionReturnSelf.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace Hostnet\Component\EntityPlugin\Fixtures;
+
+class ReflectionReturnSelf
+{
+    public function docBlock(): self
```

Comment:
IF this is valid now, then what is meant in src/ReflectionMethod.php  with "// Self is not valid when used in different places."

Reply:
I test that the function still returns null, something that was untested before